package.json

```diff
@@ -53,8 +55,41 @@
         "command": "ocaml.open-terminal-select",
         "category": "OCaml",
         "title": "Open a terminal (select a sandbox)"
+      },
+      {
+        "command": "ocaml.switch-impl-intf",
+        "category": "OCaml",
+        "title": "Switch implementation/interface"
       }
     ],
+    "keybindings": [
+      {
+        "command": "ocaml.switch-impl-intf",
+        "key": "Alt+O",
+        "when": "resourceExtname == .ml || resourceExtname == .mll || resourceExtname == .mly || resourceExtname == .re"
+      },
+      {
+        "command": "ocaml.switch-impl-intf",
+        "key": "Alt+O",
+        "when": "resourceExtname == .mli || resourceExtname == .rei"
+      }
+    ],
+    "menus": {
+      "editor/title": [
+        {
+          "command": "ocaml.switch-impl-intf",
+          "key": "Alt+O",
+          "when": "resourceExtname == .ml || resourceExtname == .mll || resourceExtname == .mly || resourceExtname == .re",
```

Comment:
Why can't we use `editorLangId` instead?

Reply:
I was only testing with the fallback implementation and made this according to `Fallback.impl` and `Fallback.intf`.

It didn't occur to me at the time that users can have files that don't match those extensions. Will fix.